src/sssom/parsers.py

```diff
@@ -607,7 +607,9 @@ def from_obographs(
                                     mdict[SUBJECT_ID] = safe_compress(nid, converter)
                                     mdict[OBJECT_ID] = safe_compress(xref_id, converter)
                                     mdict[SUBJECT_LABEL] = label
-                                    mdict[PREDICATE_ID] = "oboInOwl:hasDbXref"
+                                    mdict[PREDICATE_ID] = converter.compress(
```

Comment:
might save a bit of time to store this as a variable at the beginning of the function rather than running compress over and over